recipes/ocp/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "ocp" %}
+{% set version = "7.5.3" %}
+{% set occt_ver = "7.5.3" %}
+{% set ocp_tweak = "0" %}
+
+package:
+  name: {{ name }}
```

Comment:
looks like this is the only place `name` gets used, so you can just get rid of that variable

Reply:
I ended up using `{{ name }}` in the license file as well, so I kept this.